pkg/noun/manage.c

```diff
@@ -1027,12 +1027,31 @@ u3m_flog(c3_w gof_w)
 /* u3m_water(): produce watermarks.
 */
 void
-u3m_water(c3_w* low_w, c3_w* hig_w)
+u3m_water(c3_w* low_p, c3_w* hig_p)
```

Comment:
Why the switch from `uint32_t` to `uintptr_t`?

Reply:
These are posts aka c3_w

Reply:
Woops, my mistake - it was the `_p` suffix that caused me to misread. Now that these are going to be `u3_post`, I believe they shouldn't have a suffix at all.

Reply:
no, posts _do_ have a suffix e.g. hat_p

Reply:
That's kinda weird. I get it, since it's actually shadowing a `c3` type, but it's confusing because it's the only `u3` structure that does this (that I can think of), and it clobbers a `c3` suffix. This should probably be noted in `u3.md` - there's even a section on `u3_post` that doesn't mention this.

Reply:
It is kind of weird. `c3_p` and `c3_ps` are unused, I think, and should probably just be removed. `_p` is also incorrectly used for `void*` in a few places -- it should always be `u3_post`.